src/OrbitGl/ButtonTest.cpp

```diff
@@ -48,4 +48,17 @@ TEST(Button, SizeGettersAndSettersWork) {
   EXPECT_EQ(button.GetSize(), size);
 }
 
+TEST(Button, SizeCannotBeZero) {
```

Comment:
Consider renaming this test, maybe MinimumButtonSize?

Reply:
Hm we don't really have a consistent naming scheme for tests, do we? I prefer to name the test descriptively (e.g. the name already tells you what the test is supposed to ensure - see all other tests in this file). But I don't think I'm even doing that everywhere myself...